### ISSUE TITLE
[Adaptive Cards iOS - Input with Validation]: Placeholder text should be defined for the edit fields under input form with labels screen.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7419]
https://microsoft.visualstudio.com/OS/_workitems/edit/33769723
